src/index.ts

```diff
@@ -62,8 +62,9 @@ async function run(): Promise<PullSecretData | undefined> {
 
     await Deploy.newApp(appName, image, namespaceArg);
 
-    // Make sure the app port is exposed
-    await Deploy.patchSvc(appName, port, namespaceArg);
+    if (port) {
```

Comment:
it doesn't matter here but I prefer the explicit null check for numbers (`port != null`) because `0 == false` always seems a little off to me.